CMakeLists.txt

```diff
@@ -227,7 +227,6 @@ endif()
 
 option(WITH_SPDK "Enable SPDK" OFF)
 if(WITH_SPDK)
```

Comment:
@liupan1111 
Please add `-DWITH_SPDK=OFF \` to do_freebsd.sh
That'll save me from an extra commit